src/client/pages/Afis/AfisBetaalVoorkeuren.tsx

```diff
@@ -0,0 +1,137 @@
+import { Datalist } from '../../components/Datalist/Datalist';
+import styles from './AfisBetaalVoorkeuren.module.scss';
+import { useAppStateBagApi, useAppStateGetter } from '../../hooks/useAppState';
+import { BFFApiUrls } from '../../config/api';
+import { BagThemas } from '../../config/thema';
+import { AppRoutes } from '../../../universal/config/routes';
+import {
+  ErrorAlert,
+  OverviewPage,
+  PageHeading,
+  ThemaIcon,
+} from '../../components';
+import { isError } from '../../../universal/helpers/api';
+
+type BusinessPartnerKey = keyof Omit<
+  AfisBusinessPartnerDetailsTransformed,
+  'addressId'
+>;
+
+const labels: Record<BusinessPartnerKey, string> = {
+  businessPartnerId: 'Debiteurnummer',
+  fullName: 'Debiteurnaam',
+  address: 'Adres',
+  email: 'E-mailadres factuur',
+  phone: 'Telefoonnummer',
+};
+
+type AfisBusinessPartnerProps = {
+  businesspartner: AfisBusinessPartnerDetailsTransformed;
+};
+
+export default function AfisBetaalVoorkeuren() {
+  const { AFIS } = useAppStateGetter();
+  const businessPartnerIdEncrypted = AFIS?.content?.businessPartnerIdEncrypted;
+
+  return (
+    businessPartnerIdEncrypted && (
+      <AfisBetaalVoorkeurenContent
+        businessPartnerIdEncrypted={businessPartnerIdEncrypted}
+      />
+    )
+  );
+}
+
+function AfisBetaalVoorkeurenContent({
+  businessPartnerIdEncrypted,
+}: {
+  businessPartnerIdEncrypted: string;
+}) {
+  const [businesspartner, api] =
+    useAppStateBagApi<AfisBusinessPartnerDetailsTransformed | null>({
+      url: `${BFFApiUrls.AFIS_BUSINESSPARTNER}/${businessPartnerIdEncrypted}`,
+      bagThema: BagThemas.AFIS,
+      key: businessPartnerIdEncrypted,
+    });
+
+  const hasError = isError(api.data);
+
+  return (
+    <OverviewPage>
+      <PageHeading
+        icon={<ThemaIcon />}
+        backLink={{
+          title: 'Terug naar Afis',
+          to: AppRoutes.AFIS,
+        }}
+      >
+        Betaalvoorkeuren
+      </PageHeading>
+      <Screen>
+        <Grid>
+          <Grid.Cell span="all">
+            <Paragraph>
+              Hier kunt u uw betaal gegevens inzien en automatische incasso
+              gegevens instellen.
+            </Paragraph>
+          </Grid.Cell>
+          <Grid.Cell span="all">
+            <LinkList>
+              <LinkList.Link
+                rel="noreferrer"
+                href={
+                  'https://www.amsterdam.nl/veelgevraagd/facturen-van-de-gemeente-controleren-gegevens-wijzigen-automatische-incasso-regelen-38caa'
+                }
+              >
+                Meer over betalen aan de gemeente
+              </LinkList.Link>
+            </LinkList>
+          </Grid.Cell>
+          {hasError && (
+            <Grid.Cell span="all">
+              <ErrorAlert>
+                We kunnen op dit moment niet alle gegevens tonen.
+              </ErrorAlert>
+            </Grid.Cell>
+          )}
+          <Grid.Cell as={'section'} span="all">
```

Comment:
```suggestion
          <Grid.Cell span="all">
```
Is niet nodig om dit als Section te renderen toch?